vmdb/app/models/ems_kubernetes.rb

```diff
@@ -1,8 +1,4 @@
 class EmsKubernetes < EmsContainer
-  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
We designed these relationship to work also with amazon (groups for sure, services most likely). If we move these to the base class it's easy to (later) discover the problematic ones (if any), on the other hand if we keep them in the subclass it will be much harder to realize that kubernetes and amazon may have some relationship in common that can be moved to the base.